package exceptionss.throw1;

public class MainSite {
public static void main(String[] args) {
	System.out.println("Program started");
	Site s = new Site();
	try {
		s.login(1);
	}
	catch(NullPointerException e){
		System.out.println(e);
	}
	System.out.println("Program end");
}
}
